libero/src/main/groovy/com/connexta/libero/Config.groovy

```diff
@@ -38,8 +45,8 @@ class Config {
         mavenRepo = project.maven.repo
         quickBuild = project.maven.quickbuild
```

Comment:
Sounds good. I'm almost done with my unit tests so I should be able to merge that in soon.